Fixed LaTeX, pretty, MathML Presentation, and str printing of UniversalSet

oscargus opened the pull request:
<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests . Please also
write a comment on that issue linking back to this pull request once it is
open. -->

Fixes the printing part of #16337 

#### Brief description of what is fixed or changed
`S.UniversalSet` now prints without `()` for LaTeX and pretty.

#### Other comments


#### Release Notes

<!-- Write the release notes for this release below. See
https://github.com/sympy/sympy/wiki/Writing-Release-Notes for more information
on how to write release notes. The bot will check your release notes
automatically to see if they are formatted correctly. -->

<!-- BEGIN RELEASE NOTES -->
* printing
   * Added printing of `UniversalSet()` for LaTeX ,pretty, MathML presentation, and str printers.
<!-- END RELEASE NOTES -->

oscargus committed:
Fixed LaTeX and pretty printing of UniversalSet

sylee957 commented:
Would there be any good notation for universal set in LaTeX?
eg https://math.stackexchange.com/questions/852683/correct-set-theory-notations

oscargus commented:
Based on https://github.com/sympy/sympy/issues/16337#issuecomment-482188655 no. U, V \Xi, \epsilon seems to be options, but none universal. "but it might as well be any other letter"

smichr commented:
> but it might as well be any other letter

The [double struck U looks nice](https://www.symbols.com/symbol/universal%20set) and there is a motivation for using it presented [here](https://www.answers.com/Q/What_is_the_symbol_for_universal_set).

jksuom commented:
I think that [blackboard bold](https://en.wikipedia.org/wiki/Blackboard_bold) could be used. The LaTeX command (in math mode) is `\mathbb` when `amsfonts` has been loaded. (That should be in the mathjax config file TeX-AMS_HTML-full.)

oscargus commented:
I thought that this would be the least controversial PR ever, just a formality, but it turns out to probably be the most discussed. ;-)

I've asked @asmeurer to install https://gh-polls.com/ so that we can have a poll. In the meanwhile I will prepare renderings of the alternatives...

asmeurer commented:
Hmm, OK. I think I've enabled it. I'd rather not have too many things that have API access to the SymPy repo hanging around, but let's see how this works.

smichr commented:
I don't think we need a poll. @sylee957 asked a question, I gave a motivation for picking "U" and a typeface for it (which @jksuom seconded). Can we just go with blackboard U... as in "UniversalSet", "Union of all sets" and "Uber set"? We would never have to worry about seeing `UniversalSet U otherset` since this would reduce to (blackboard) U, right?

my 2c

oscargus committed:
Changed output and added MathML presentation and str printing

oscargus commented:
@smichr you are right. I initially had the impression that there were different opinions.

I have changed accordingly (and added MathML presentation and str-printing), the current output is:
![image](https://user-images.githubusercontent.com/8114497/56430249-195bbd00-62c6-11e9-9899-fa7358df05ae.png)

oscargus committed:
Fixed DocTests

asmeurer commented:
I removed the polls app. Like I said, I don't like having too many random apps with API access to the repo. I also personally don't think polls are the right way to do decision making for SymPy. Generally things should be based on consensus, meaning we discuss things until we agree on something. That often requires making concessions. If people can't come to an agreement we may have to resort to something else. In the worst case I use my BDFL powers to force a decision (this is very rare). 

I also like the blackboard U for the universal set FWIW. I think it's fairly self evident what it means, and I'm not aware of any other uses of blackboard U.

oscargus commented:
I agree that consensus is better than polls. As this was a rather non-critical issue (to some extent) a poll might have worked though. Still, I didn't understand how to use it, so no harm from my perspective.
